src/hindi.js

```diff
@@ -10,27 +10,27 @@ class Hindi {
 
   conjugate (word, info) {
     // Format for info:
-    // {tense: 'Past/Present/Future', gender: 'male/female'}
+    // {tense: 'past/present/future', gender: 'male/female'}
     word = word.trim().toLowerCase();
 
-    switch (info.tense) {
-      case 'Present':
-        if (info.gender === 'male') {
+    switch (info.tense.toLowerCase()) {
+      case 'present':
+        if (info.gender && info.gender.toLowerCase() === 'male') {
           return word.replace(/na$/, 'ta hun');
         }
         return word.replace(/na$/, 'ti hun');
-      case 'Past':
-        if (info.gender === 'male') {
+      case 'past':
+        if (info.gender && info.gender.toLowerCase() === 'male') {
```

Comment:
Why && info.gender.toLowerCase() and not just:
info.gender.toLowerCase() === 'male' ?